src/aiida/tools/pytest_fixtures/daemon.py

```diff
@@ -60,11 +62,61 @@ def test(started_daemon_client):
         daemon_client.start_daemon()
         assert daemon_client.is_daemon_running
 
+    import threading
+    import time
+
+    from watchdog.events import FileSystemEventHandler
+    from watchdog.observers import Observer
+
+    logger = logging.getLogger('tests.daemon:started_daemon_client')
+
+    logger.debug(f'Daemon log file is located at: {daemon_client.daemon_log_file}')
```

Comment:
Which log level do I have to change to debug to see this message? It tried to run `pytest -s --pdb` but I don't see this message

Reply:
You need to run `pytest tests --log-cli-level=DEBUG` as mentioned in the description of PR.

Reply:
The way to reproduce the write to the daemon log is to run test `tests/engine/processes/test_control.py::test_pause_processes -v --log-cli-level=DEBUG -s` with print some text in plumpy's `Process::pause` method. The reason that the print would show things to the REPL where pytest is running since it calls from a rpc call and runs in a daemon worker. The print output will be then written to the daemon log.

Reply:
Since you encounter a problem that seems when the RMQ not start, the daemon failed to start and the daemon log is written but not dump to debug message, I'll revert the change after. It still help quite a lot by printing the daemon log path so we can go the check manually. What do you think?

Reply:
It also does not work for me with rabbitmq, I am running
```
pytest --maxfail=1 -s --log-cli-level=DEBUG tests/engine/processes/test_control.py -k test_pause_processes
```
I am checking the prints in `plumpy/processes.py::Process.paused`
```python
    @property
    def paused(self) -> bool:
        """Return whether the process was being paused."""
        print("PAUSE")
        return self._paused is not None
```
I see the PAUSE in the log but not printed to the screen.